pkg/index/index/index.go

```diff
@@ -63,6 +63,8 @@ type CosignSignature struct {
 	CertificateIdentity         string `yaml:"certificate-identity"`
 	CertificateIdentityRegexp   string `yaml:"certificate-identity-regexp"`
 	CertificateGithubWorkflow   string `yaml:"certificate-github-workflow"`
+	KeyRef                      string `yaml:"key"`
+	PrivateInfrastructure       bool   `yaml:"private-infrastructure"`
```

Comment:
this should be the same name as the struct `IgnoreTlog`

```suggestion
	IgnoreTlog       bool   `yaml:"ignore-tlog"`
```